tests/components/mqtt/test_device_tracker.py

```diff
@@ -429,6 +429,84 @@ async def test_setting_device_tracker_location_via_lat_lon_message(
     assert state.state == STATE_UNKNOWN
 
 
+async def test_setting_device_tracker_location_via_auto_zone_message(
+    hass, mqtt_mock_entry_no_yaml_config, caplog
+):
+    """Test the setting of the latitude and longitude via MQTT."""
+    await mqtt_mock_entry_no_yaml_config()
+    async_fire_mqtt_message(
+        hass,
+        "homeassistant/device_tracker/bla/config",
+        "{ "
+        '"name": "test", '
+        '"state_topic": "test-topic", '
+        '"json_attributes_topic": "attributes-topic" '
+        "}",
```

Comment:
I would the special payload `pl_aut_zn` to be set here

Reply:
Add another test? Sorry, what do you mean exactly?

Reply:
I could add an explicit "setting" here that sets the `payload_auto_zone` to another custom payload (like "auto"), in the same test, is that what you're asking?

Reply:
> Add another test? Sorry, what do you mean exactly?

A test that asserts you can specify a custom payload. Then you could it leave as default in this one.

Reply:
Ill make another test that'll use the abbreviation 👍